pkg/lib/config/validate.go

```diff
@@ -0,0 +1,26 @@
+package config
+
+import (
+	"github.com/operator-framework/operator-registry/pkg/declcfg"
+)
+
+// ValidateConfig takes a directory containing the declarative config file(s)
+// 1. Validate if declarative config file(s) are valid based on specified schema
+// 2. Validate the `replaces` chains of the upgrade graph
+// Inputs:
+// directory: the directory where declarative config file(s) exist
+// Outputs:
+// error: ValidattionError which contains a list of errors
+func ValidateConfig(directory string) error {
+	// Load config files
+	cfg, err := declcfg.LoadDir(directory)
+	if err != nil {
+		return err
+	}
+	// Validate the config using model validation
```

Comment:
I'm guessing this is not ready to be reviewed yet?

Reply:
I just rebased and that's pretty much it. The convert func will do validation in the background so that's all we need for validation at this point.

Reply:
Just driving by as I was curious - can we reflect that we're relying on that ConvertToModel function to implicitly validate as it's not super clear with either the current comment or the current ValidateConfig function comments.

Reply:
Sure. I will comment on that to make it more transparent.